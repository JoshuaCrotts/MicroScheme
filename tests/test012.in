(define pi 3.141592685)
(define r 5)
(define circumference (* 2 pi r))
circumference
