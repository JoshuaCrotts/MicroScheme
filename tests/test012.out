31.415926850
